hive-data-gen/src/main/java/com/wipro/ats/bdre/datagen/util/TableUtil.java

```diff
@@ -72,10 +72,11 @@ public String getDelimitedTextRow(Table table, String pid) throws IOException {
 
 
         List<Column> columns = table.getColumns();
-        int i = 0;
         StringBuilder rowData = new StringBuilder();
         for (Column column : columns) {
-            if (column.getRefTable() != null) break;
+            if (column.getRefTable() != null) {
+                break;
+            }
             rowData.append(getData(column.getColumnGenId(), pid));
             //LOGGER.debug("Table="+table.getTableName()+"; column="+column.getColumnName()+"; index="+column.getColumnIndex());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)